.github/etc/dictionary.txt

```diff
@@ -1,2 +1,5 @@
 capricornis
 crispus
+kubeconform
```

Comment:
Consider adding a newline at the end of the file for consistency.

Reply:
Success!  🎉